packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageReadStatusIndicatorStyles.ts

```diff
@@ -0,0 +1,19 @@
+import { ICSSInJSStyle, ComponentSlotStylesPrepared } from '@fluentui/styles';
+import { ChatMessageReadStatusIndicatorStylesProps } from '../../../../components/Chat/ChatMessageReadStatusIndicator';
+import { screenReaderContainerStyles } from '../../../../utils';
+import { ChatMessageReadStatusIndicatorVariables } from './chatMessageReadStatusIndicatorVariables';
+
+export const ChatMessageReadStatusIndicatorStyles: ComponentSlotStylesPrepared<
+  ChatMessageReadStatusIndicatorStylesProps,
+  ChatMessageReadStatusIndicatorVariables
+> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    position: 'absolute',
```

Comment:
Did you check if this is the best approach from accessibility point of view? Usually we avoid `absolute` positioned elements if it is not really the only way to go

Reply:
Tried with screen reader on mac and it works as expected appending the `title` to the end of the message, couldn't think in another way to achieve the same ( I am basically replicating the behavior of `author` here )

Reply:
Perfect, disregard my comment then :)